locales/tr.json

```diff
@@ -1,304 +1,304 @@
       },
       {
-        "title": "Can I trust Dumpus?",
-        "description": "This \"Data Package\" is very sensitive. That's why anyone can read our code and see what we do with it (100% open source). Although we offer a free processing server, you can use your own server to keep full control over your data."
+        "title": "Dumpus'a güvenebilir miyim?",
+        "description": "Bu \"Veri Paketi\" oldukça hassastır. Bu yüzden herkes yazılımımızın (%100 açık) kaynak kodlarını okuyup bu paketle ne yaptığımızı görebilir. Ücretsiz bir veri işleme sunucusu sunsakta kendi sunucu kullanarak verini tamamen kontrol altına alabilirsin."
       },
       {
-        "title": "Share it with your friends!",
-        "description": "While statistics are funny to see, they are even more fun to compare with your friends. That is why we have made it easy to share your stats with your friends."
+        "title": "Arkadaşlarınla paylaş!",
+        "description": "İstatikleri görmek eğlenceli ve komik olsada, arkadaşlarınla karşılaştırmak daha eğlencelidir. Bu yüzden istatiklerini arkadaşlarınla paylaşabilmeni kolaylaştırdık."
       }
     ],
     "setup": {
-      "title": "How do I get my data package?",
-      "description": "Select an operating system to get started.",
+      "title": "Veri paketime nasıl erişebilirim?",
+      "description": "Başlamak için bir işletme sistemi seç.",
       "osDisplays": {
         "android": "Android",
         "ios": "iOS",
-        "desktop": "Desktop"
+        "desktop": "Masaüstü"
       },
       "steps": {
-        "settings": "Open your Discord settings",
-        "privacy": "Open the \"Privacy & Safety\" tab",
-        "request": "Click on \"Request Data\""
+        "settings": "Discord ayarlarını aç",
+        "privacy": "\"Gizlilik ve Güvenlik\" sekmesini aç",
+        "request": "\"Tüm Verilerimi iste\" düğmesine tıkla"
       }
     },
     "access": {
-      "title": "Submit your Data Package",
-      "description": "Choose your favorite method.",
-      "method": "Method {{n}}",
-      "noPackageYet": "No package yet?",
+      "title": "Veri Paketini gönder",
+      "description": "Favori yöntemini seç.",
+      "method": "Yöntem {{n}}",
+      "noPackageYet": "Henüz paketin yok mu?",
       "methods": {
         "link": {
-          "title": "Enter your Discord link",
-          "description": "You can get it in the email received from Discord. Copy the link from the button entitled **You can download it here**.",
+          "title": "Discord linkini gir",
+          "description": "Discord tarafından gönderilmiş mail tarafından alabilirsin. **You can download it here** yazan düğmeden linki kopyalabilirsin.",
           "discordLink": "Discord link",
-          "advanced": "Advanced",
+          "advanced": "Gelişmiş",
           "backendURL": "Backend URL",
```

Comment:
Backend URL doesn't have a proper way to translate to Turkish, so I left it this way.